code/modules/religion/burdened/psyker.dm

```diff
@@ -0,0 +1,361 @@
+/obj/item/organ/internal/brain/psyker
+	name = "psyker brain"
+	desc = "This brain is blue, split into two hemispheres, and has immense psychic powers. What kind of monstrosity would use that?"
+	icon_state = "brain-psyker"
+	actions_types = list(
+		/datum/action/cooldown/spell/pointed/psychic_projection,
+		/datum/action/cooldown/spell/charged/psychic_booster,
+		/datum/action/cooldown/spell/forcewall/psychic_wall,
+	)
+	organ_traits = list(TRAIT_ANTIMAGIC_NO_SELFBLOCK)
+
+/obj/item/organ/internal/brain/psyker/Insert(mob/living/carbon/inserted_into, special, drop_if_replaced, no_id_transfer)
+	if(!istype(inserted_into.get_bodypart(BODY_ZONE_HEAD), /obj/item/bodypart/head/psyker))
+		return FALSE
+	. = ..()
```

Comment:
what happens if your head gets augmented later?

since the blindness is now tied to the echolocation directly, it might be fine to just remove the "psyker brain must be in a psyker head" restriction so that you don't have to account for all of these edge cases.